src/test/scala/no/gdl/bookapi/model/domain/TranslationTest.scala

```diff
@@ -69,6 +69,70 @@ class TranslationTest extends IntegrationSuite with TestEnvironment {
     }
   }
 
+  test("that bookIdsWithLanguageAndLevel returns ids with same language when level is None") {
+    withRollback { implicit  session =>
+      val book1 = addBookDef()
+      val book2 = addBookDef()
+      val book3 = addBookDef()
+
+      val translationForBook1 = addTranslationDef("ext1", "title 1", book1.id.get, "xho", None)
+      val translationForBook2 = addTranslationDef("ext2", "title 2", book2.id.get, "xho", None)
+      val translationForBook3 = addTranslationDef("ext3", "title 3", book3.id.get, "xho", None)
+
+      val ids = Translation.bookIdsWithLanguageAndLevel("xho", None, 10, 1)
+      ids.length should be(3)
+      ids.min should equal (book1.id.get)
+      ids.max should equal (book3.id.get)
+    }
+  }
+
+  test("that bookIdsWithLanguageAndLevel only returns ids for same level") {
+    withRollback { implicit session =>
+      val book1 = addBookDef()
+      val book2 = addBookDef()
+      val book3 = addBookDef()
+
+      val translationForBook1 = addTranslationDef("ext1", "title 1", book1.id.get, "xho", Some("2"))
+      val translationForBook2 = addTranslationDef("ext2", "title 2", book2.id.get, "xho", Some("2"))
+      val translationForBook3 = addTranslationDef("ext3", "title 3", book3.id.get, "xho", Some("1"))
+
+      val ids = Translation.bookIdsWithLanguageAndLevel("xho", Some("2"), 10, 1)
+      ids.length should be(2)
+      ids.min should equal (book1.id.get)
+      ids.max should equal (book2.id.get)
```

Comment:
Can be replaced with `ids shouldEqual Seq(book1, book2).map(_.id.get)`? (again, only if the order doesn't matter)